### PR TITLE
Tech note about our newsletter workflow

### PR DESCRIPTION
**Pull request by maelle**
* [x] I have read the Content Guidelines.
* [x] I have read the Technical Guidelines.
* [x] I used or followed the R Markdown or Markdown template.
* [x] I have followed the Style Guide.
* [x] I created or updated my author metadata with correct folder name.
* [x] I have added relevant tags after browsing existing tags (incuding "community" tag).
* [x] I have added the "tech notes" tag if this is a technote.
* [x] I ran `roblog::ro_lint_md()` on index.md (optional).
* [x] I ran `roblog::ro_check_urls()` on index.md (optional).
* [x] I ran a spell-check on index.md.

### CODE REVIEW

File: content/blog/2021-06-28-news-meta/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+To populate this, we use information from the website (comm calls), our reading (was an interesting answer or question sent to the R-package-devel mailing list?), etc.
+To not forget ideas or to bring them to the attention of the newsletter curator we store them as issues of a GitHub repository.
+
+Furthermore, the manually curated sections are reviewed by at least one person more.
+The posts don't get a [full blown review](https://blogguide.ropensci.org/) but this helps strengthen them.
+
+### Newsletters as emails: hello Sendgrid, bye MailChimp!
+
+We used to send our newsletter via MailChimp, however we heard [bad things about the company itself](https://www.businessinsider.fr/us/inside-mailchimp-mass-exodus-women-people-color-ben-chestnut-2021-3) so asked around for recommendations (thanks a ton to everyone who chimed in!) and settled on SendGrid whose email service we were actually already using for the [CRAN checks API](https://blog.r-hub.io/2019/06/10/cran-checks-api/) and our forum.
+So, now, our newsletter is emailed via SendGrid **Marketing** service.[^transfer]
```
Comment (steffilazerte) [inline content/blog/2021-06-28-news-meta/index.Rmd:101]: Why bold the "Marketing"?

File: content/blog/2021-06-28-news-meta/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+How do we go about that?
+We extract the HTML corresponding to the post from our [full content RSS feed](/rbloggers/index.xml) (set up for R Bloggers).
+We use that feed as all relative links (except for images) have been made absolute at that point.
+We copy-paste it into a text editor and make a few tweaks:
+
+* make the images src an absolute link,
+* removing all heading anchors,
```
Comment (steffilazerte) [inline content/blog/2021-06-28-news-meta/index.Rmd:109]:
```suggestion
* remove all heading anchors, 
```

File: content/blog/2021-06-28-news-meta/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+So, now, our newsletter is emailed via SendGrid **Marketing** service.[^transfer]
+
+How do we go about that?
+We extract the HTML corresponding to the post from our [full content RSS feed](/rbloggers/index.xml) (set up for R Bloggers).
+We use that feed as all relative links (except for images) have been made absolute at that point.
+We copy-paste it into a text editor and make a few tweaks:
+
+* make the images src an absolute link,
+* removing all heading anchors, 
+* adding a [no tracking attribute to links](https://community.auth0.com/t/howto-disable-sendgrids-click-tracking-feature-in-an-auth0-email-template/22958) as the global SendGrid account option didn't work for that.[^tracking]
```
Comment (steffilazerte) [inline content/blog/2021-06-28-news-meta/index.Rmd:110]:
```suggestion
* add a [no tracking attribute to links](https://community.auth0.com/t/howto-disable-sendgrids-click-tracking-feature-in-an-auth0-email-template/22958) as the global SendGrid account option didn't work for that.[^tracking]
```

File: content/blog/2021-06-28-news-meta/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+Voilà, at that point all that remains is tweeting about the newsletter, and hoping it gets read at least a bit.
+
+Note that SendGrid sign-up forms offer much less flexibility than MailChimp similar features, so we had to add an iframe to our website to incorporate sign-up, where we used to have a more integrated form whose JS code pinged a dedicated MailChimp URL.
+
+## Conclusion
+
+In this post we presented our workflow for curating and sending our monthly newsletter.
+Our newsletter is published as a monthly post on our blog, and sent as an email to subscribers via SendGrid.
+We prepare its content with a mix of automatically generated and manually curated sections, followed by a review for the latter.
+The automatical generation of some content depends on upstream manual and automatic work for gathering data (e.g. our packages registry, citations monitoring, encouraging users to report use cases).
```
Comment (steffilazerte) [inline content/blog/2021-06-28-news-meta/index.Rmd:130]:
```suggestion
The automatic generation of some content depends on upstream manual and automatic work for gathering data (e.g. our packages registry, citations monitoring, encouraging users to report use cases).
```

**Review by steffilazerte — COMMENTED** [review]
Hi @maelle, generally looks good! I have some wording suggestion, and we just need a bit more YAML details and a publication date. @stefaniebutland suggested soon, Tues or Wed this week, so just let me know when you're ready. I'll draft a tweet tomorrow morning.

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [ ] publication date is ok
* [X] alternative text of images is informative
* [ ] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
    * **@maelle, do you want to update the Description and Draft Tweet Text?**
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)

File: content/blog/2021-06-28-news-meta/index.Rmd (hunk 1 above)
Comment (maelle) [reply content/blog/2021-06-28-news-meta/index.Rmd:101]: I'll re-phrase. SendGrid has both email and marketing plans and it was not obvious to me at first that we were after the marketing part. :sweat_smile:

**Pull request merged**